helm/kubernetes-scanner/values.yaml

```diff
@@ -168,11 +168,17 @@ config:
 
 # MANDATORY: The name of a secret containing auth credentials, which must be
 # provisioned outside of this chart.
+#
 # Requires a key "snykServiceAccountToken". You can find more information on
 # Snyk service accounts at https://docs.snyk.io/snyk-admin/service-accounts.
 # The service account must be associated with the organizationID configured
 # above.
 #
+# The service account must have one of the following roles:
+# - Org Admin
+# - Group Admin
+# - Custom Role with "Publish Kubernetes Resources" permission
```

Comment:
I'm a bit confused here. The group-level service-accounts don't have granular permissions, right?
Where can I actually define permissions for an org-level service account? 🤔 In my org, the only role I can give to a service account is "Org admin".

Or was it group-level SAs that can have more granular permissions? 🤔 

(I'm asking because I initially wondered if customers can define these permissions themselves or if a customer success person needs to do that)

Reply:
I think you have to create the custom role first for it to be visible as a service account role: https://docs.snyk.io/snyk-admin/manage-users-and-permissions/member-roles

I also heard that group-level custom roles don't exist yet, but that's orthogonal to this config I reckon.

Reply:
Ah okay, that makes sense. "custom role" in the sense of "you need to create it beforehand", obviously 😄 

I somehow expected this to exist already 🤦‍♂️